packages/net-www/exlibs/vivaldi.exlib

```diff
@@ -73,15 +73,9 @@ vivaldi_src_install() {
     edo mv opt "${IMAGE}/"
     dodir /usr/share
     edo mv usr/share "${IMAGE}/usr/"
-
-    herebin ${PN%-*} << EOF
-#!/bin/sh
-exec /opt/${PN%-stable}/vivaldi "\$@"
-EOF
-
-    einfo "You may experience issues with running ${PN} on some linux configurations (kernel lower"
-    einfo "than 3.17 or disabled \"user namespaces\") without suid bit on vivaldi-sandbox."
-    einfo "See next link:"
-    einfo "https://vivaldi.net/teamblog/79-sandbox-issues-when-installing-to-non-standard-locations-on-linux"
-EOF
+    edo rm -rf "${IMAGE}usr/share/gnome-control-center/"
+    dodir /usr/$(exhost --target)/bin
+    dodir /usr/share/pixmaps
+    dosym /opt/${PN%-stable}/product_logo_256.png /usr/share/pixmaps/${PN}.png
+    dosym /opt/${PN%-stable}/${PN%-stable} /usr/$(exhost --target)/bin/${PN}
```

Comment:
I suspect that `${PN%-stable}` will not be effective for `PN=vivaldi-snapshot`.

Original script were using this only for folder name which resulted in `exec /opt/vivaldi/vivaldi "$@"` and `exec /opt/vivaldi-snapshot/vivaldi "$@"`. But here we make a symlink with name `vivaldi-snapshot` to `/opt/vivaldi-snapshot/vivaldi-snapshot` which I guess is wrong.

Beside that, I'm not sure if it is intended to have `vivaldi-stable` which previously was just `vivaldi`.

Reply:
I tested both packages locally, and that's what worked. If package name doesn't have `-stable` it will keep whatever it has, in this case `-snapshot`, that was my intention.